Traceback (most recent call last):
  File "/scratch/tyoeasley/brain_representations/src_py/run_comp_Psim.py", line 4, in <module>
    cpm.comp_dist_mtx(sys.argv[1],sys.argv[2])
  File "/mnt/beegfs/scratch/tyoeasley/brain_representations/src_py/comp_Psim_mtx.py", line 14, in comp_dist_mtx
    Psim_mtx = comp_Psim_vals(subj_list)
  File "/mnt/beegfs/scratch/tyoeasley/brain_representations/src_py/comp_Psim_mtx.py", line 45, in comp_Psim_vals
    data_list[i] = parse.parse_fname(subj_list[i])
  File "/mnt/beegfs/scratch/tyoeasley/brain_representations/src_py/parse_neurodata.py", line 12, in parse_fname
    data=parser(fname)
  File "/mnt/beegfs/scratch/tyoeasley/brain_representations/src_py/parse_neurodata.py", line 30, in load_csv
    data0 = np.genfromtxt(fname,delimiter=",")
  File "/home/e.ty/.conda/envs/alg_top_neuro/lib/python3.7/site-packages/numpy/lib/npyio.py", line 1759, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "/home/e.ty/.conda/envs/alg_top_neuro/lib/python3.7/site-packages/numpy/lib/_datasource.py", line 269, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/e.ty/.conda/envs/alg_top_neuro/lib/python3.7/site-packages/numpy/lib/_datasource.py", line 623, in open
    raise IOError("%s not found." % path)
OSError: subj-100206_catrun.csv not found.
